CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 <!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
 * Removed beta tags for TeamProjectAccess by @rberecka [#756](https://github.com/hashicorp/go-tfe/pull/756)
 
+## Enhancements
+* Added BETA support for including `exclusions` relationship to policy_set on create by @Netra2104 [#757](https://github.com/hashicorp/go-tfe/pull/757)
```

Comment:
perhaps a mention that currently only workspaces can be excluded?

Reply:
Is this to be a polymorphic relationship?

Reply:
A policy set has relationship with many workspaces. This is similar in a sense that a policy set has a relationship with a list of workspaces that will be excluded during a run.